### PR TITLE
hack/usoc22: Update session 02: Behind the Scenes

### PR DESCRIPTION
**Pull request by razvand**
Update contents for session 02: Behind the Scenes, together with USoC22 structure.

### CODE REVIEW

File: content/en/community/hackathons/usoc22/behind-scenes/index.md [hunk 1]
```diff
@@ -100,4 +100,5 @@ weigth: 20
 
 ### 12. Give Us Feedback
 
-{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_12.-Give-Us-Feedback.md" markdown="true" >}}
+We want to know how to make the next sessions better.
+Fo this we need your [feedback](https://docs.google.com/forms/d/e/1FAIpQLSdpyEpCW0yxJBSDa7Iq95JUCe5YNnRpxOD31ol4ezpKWnqrhA/viewform).
```
Comment (razvanvirtan) [inline content/en/community/hackathons/usoc22/behind-scenes/index.md:104]:
```suggestion
For this we need your [feedback](https://docs.google.com/forms/d/e/1FAIpQLSdpyEpCW0yxJBSDa7Iq95JUCe5YNnRpxOD31ol4ezpKWnqrhA/viewform).
```
My bad, I missed this in the patch file.

**Review by razvanvirtan — CHANGES_REQUESTED** [review]

File: content/en/community/hackathons/usoc22/behind-scenes/index.md (hunk 1 above)
Comment (razvand) [reply content/en/community/hackathons/usoc22/behind-scenes/index.md]: fixed

**Review by razvanvirtan — APPROVED** [review]
All good on my side.
Reviewed-by: Razvan Virtan <virtanrazvan@gmail.com>

**Review by nderjung — APPROVED** [review]
Thanks!

Approved-by: Alexander Jung <alex@unikraft.org>

**Pull request merged**